sql3/parser/parser.go

```diff
@@ -3283,6 +3413,36 @@ func (p *Parser) parseIntegerLiteral(desc string) (*IntegerLit, error) {
 	}
 }
 
+func (p *Parser) parseAlterDatabaseStatement(alterPos Pos) (_ *AlterDatabaseStatement, err error) {
+	var stmt AlterDatabaseStatement
+	stmt.Alter = alterPos
+	if p.peek() != DATABASE {
+		return &stmt, p.errorExpected(p.pos, p.tok, "DATABASE")
+	}
+	stmt.Database, _, _ = p.scan()
+
+	if stmt.Name, err = p.parseIdent("database name"); err != nil {
+		return &stmt, err
+	}
+
+	switch p.peek() {
+	case SET:
```

Comment:
Consistency of syntax question, could we re-use the syntax of `CREATE....WITH`...the specified options being the ones to be altered - so this becomes `ALTER DATABASE foo WITH UNITS = new_units;`?